### ISSUE TITLE
isUserLoading remains true after loadUser() if no user in storage

### ISSUE THREAD

**Issue by mcosand** [maxmantz/redux-oidc#73]
I have an app that has various requirements around authentication. Some pages require authentication; unauthenticated users should automatically start the signinRedirect.

I am calling loadUser immediately after creating the store (same as the example app).
isLoadingUser gets set to true so I know that I need to wait before firing the redirect, but if no user was in storage then there's no indication that the load is finished and I don't know that it's time to do the redirect.

One of the project's existing test cases is "should dispatch USER_EXPIRED when no user is present" - is this the desired behavior? Firing that action would reset the isLoadingUser flag to false and would be good for my app. Is there any reason not to fire USER_EXPIRED in this situation? The test case uses an expired vs a null user, which I don't think correctly describes my scenario.

I can get a pull request together if USER_EXPIRED should be fired when loadUser is called with an empty storage.

**Comment by mcosand** [comment]
I managed to get this working more like I expected by skipping redux-oidc's loadUser and chaining off the getUser method. In one of the .then methods I could call redux-oidc's userExpired and/or signinSilent.

This results in some duplicate actions being fired if OidcProvider is mounted before the usermanager promises are resolved. I removed my OidcProvider and bound the userManager.events to redux-oidc actions in the last .then method.
